firmware/mcal/stm32f767/periph/can.h

```diff
@@ -83,6 +84,19 @@ class CanBase : public shared::periph::CanBase {
     static constexpr uint32_t kCanRxActiveInterruptFifo0 =
         CAN_IT_RX_FIFO0_MSG_PENDING;
 
+    static constexpr uint32_t kDefaultFilterIdHigh = 0x0000;
+    static constexpr uint32_t kDefaultFilterIdLow = 0x0000;
+    static constexpr uint32_t kDefaultFilterMaskIdHigh = 0x0000;
+    static constexpr uint32_t kDefaultFilterMaskIdLow = 0x0000;
+
+    static constexpr uint32_t kFilterScale = CAN_FILTERSCALE_32BIT;
+    static constexpr uint32_t kFilterEnable = CAN_FILTER_ENABLE;
```

Comment:
What's the purpose of renaming constants? Is it to add ownership of a value to this class?

Reply:
I believe Sam is doing this to signify a separation between the underlying cubemx driver code and our app layer code. Plus it follows the google naming standard for variables so I think this is a good change.